cli-plugins/insight/triaging-vulnerabilities.hbs.md

```diff
@@ -118,3 +118,46 @@ The following conditions are required for this action:
 > **Note** The responsibility of assessing a vulnerability's impact is up to the person in charge of
 > triage. Images and sources with the same package and version might use the
```

Comment:
Can we also include a note saying that this feature is in "Alpha"

Also, may want to move the Notes below the H2 below (or a tap docs person can fix it for us if it's a problem... I might be making that up)

Reply:
As mentioned above, the whole Triage feature is still alpha, so this should be unnecessary